THEMING.md

```diff
@@ -0,0 +1,142 @@
+<li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
+</ul>
+
+#### Container scheme
+
+The Container Scheme represents a single configurable entity that is applicable to all themeable Components. A Container Scheme consists of all of the Subsystem Schemes.
+
+## Examples
+
+### Using a subsystem scheme
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents.MaterialColorScheme
+
+let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+// Configure custom properties to match your branch
+colorScheme.backgroundColor = .black
+colorScheme.primaryColor = .yellow
+colorScheme.secondaryColor = .red
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialColorScheme.h"
+
+MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+// Configure custom properties to match your branch
+colorScheme.backgroundColor = UIColor.blackColor;
+colorScheme.primaryColor = UIColor.yellowColor;
+colorScheme.secondaryColor = UIColor.redColor;
+```
+<!--</div>-->
+
+### Creating a container scheme
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents.MaterialColorScheme
+import MaterialComponents.MaterialTypographyScheme
+import MaterialComponents.MaterialShapeScheme
+
+let containerScheme = MDCContainerScheme()
+containerScheme.colorScheme = myColorScheme
+containerScheme.typographyScheme = myTypographyScheme
+containerScheme.shapeScheme = myShapeScheme
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialColorScheme.h"
+#import "MaterialShapeScheme.h"
+#import "MaterialTypographyScheme.h"
+
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+containerScheme.colorScheme = self.myColorScheme;
+containerScheme.shapeScheme = self.myShapeScheme;
+containerScheme.typographyScheme = self.myTypographyScheme;
+```
+<!--</div>-->
+
+### Theme component
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents.MaterialButtons
+import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
+
+
+let containerScheme = MDCContainerScheme()
+let button = MDCButton()
+button.applyTextTheme(withScheme: containerScheme)
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialButtons.h"
+#import "MaterialButtons+Theming.h"
+#import "MaterialContainerScheme.h"
+
+
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+MDCButton *button = [[MDCButton alloc] init];
+[button applyTextThemeWithScheme:containerScheme];
+```
+<!--</div>-->
+
+
+### Additional links
```

Comment:
Nit: This should probably be an `h2` (`##`) since it's not part of the "Examples" section.

Reply:
Done